### PR TITLE
Prevent throwing errors for `publicData` getter when value is not loaded yet.

### PR DESCRIPTION
**Pull request by yavorsky**
### 🔦 Summary
This PR fixes our `PublicData` Provider to not throw errors when value is not loaded yet. It's pretty ok to rerender it after it's loaded with some value.

Currently if we are making something like `publicData.get('title')` and it wasn't loaded yet, we just throwing an error.

This PR propose to return `undefined` since we are going to renender it anyway when value will be loaded.

Second change related to timeout we are checking for `publidData` request was succeed. Sometimes when connection is not good, it's really doesn't fit 500ms requirements.

### CODE REVIEW

File: packages/yoshi-flow-editor-runtime/src/react/PublicDataProviderViewer.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ export class PublicDataProviderViewer extends React.Component<
     try {
       return this.props.data[scope][key];
     } catch (e) {
-      console.error(new Error(`key "${key}" wasn't found on public data:`));
```
Comment (yavorsky) [inline packages/yoshi-flow-editor-runtime/src/react/PublicDataProviderViewer.tsx:18]: Seems like it was typo :/

File: packages/yoshi-flow-editor-runtime/src/react/PublicDataProviderEditor.tsx [hunk 2]
```diff
@@ -78,7 +79,7 @@ export class PublicDataProviderEditor extends React.Component<
   handleSetParam = (key: string, value: any) => {
     const { Wix } = this.props;
 
-    if (!this.state.ready || !this.state.data) {
```
Comment (yavorsky) [inline packages/yoshi-flow-editor-runtime/src/react/PublicDataProviderEditor.tsx:81]: We are going to have data as an object anyway to prevent excess checks for `null`.

File: packages/yoshi-flow-editor-runtime/src/react/PublicDataProviderEditor.tsx [hunk 3]
```diff
@@ -68,8 +69,8 @@ export class PublicDataProviderEditor extends React.Component<
   }
 
   handleGetParam = (key: string) => {
-    if (!this.state.ready || !this.state.data) {
-      throw this.state.readyPromise;
+    if (!this.state.ready) {
+      return undefined;
```
Comment (yavorsky) [inline packages/yoshi-flow-editor-runtime/src/react/PublicDataProviderEditor.tsx:73]: We are going to return `undefined` instead of throwing an error, which is pretty ok for getter.

File: packages/yoshi-flow-editor-runtime/src/react/PublicDataProviderEditor.tsx [hunk 4]
```diff
@@ -21,17 +21,18 @@ export class PublicDataProviderEditor extends React.Component<
...
     const { Wix } = this.props;
 
     const publicDataPromise = new Promise((resolve, reject) => {
       setTimeout(function() {
-        reject(new Error('timeout of 500ms reached on Wix.Data.Public.getAll'));
-      }, 500);
+        reject(
+          new Error('timeout of 1500ms reached on Wix.Data.Public.getAll'),
+        );
+      }, 1500);
```
Comment (yavorsky) [inline packages/yoshi-flow-editor-runtime/src/react/PublicDataProviderEditor.tsx:35]: sometimes 500 ms is not enough.

File: packages/yoshi-flow-editor-runtime/src/WidgetWrapper.tsx [hunk 5]
```diff
@@ -23,7 +23,8 @@ interface IFrameworkProps {
 }
 
 // Bolt ssr environment doesn't contain window.
-const globalWix = typeof window !== 'undefined' ? window.Wix : undefined;
+const inBrowser = typeof window !== 'undefined';
```
Comment (yavorsky) [inline packages/yoshi-flow-editor-runtime/src/WidgetWrapper.tsx:26]: This change is not related to PR general idea, just small clarification for this check.

File: packages/yoshi-flow-editor-runtime/src/react/PublicDataProviderEditor.tsx (hunk 4 above)
Comment (ranyitz) [reply packages/yoshi-flow-editor-runtime/src/react/PublicDataProviderEditor.tsx:35]: I would put this timeout in a constant.
